.vscode/settings.json

```
@@ -0,0 +1,5 @@
+{
+   "prettier.tabWidth": 3,
```

Чаще всего 
```suggestion
   "prettier.tabWidth": 2,
```

Иногда могут использовать 4, но это я видел очень редко в JS/TS проектах